src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -617,3 +644,39 @@ def _parse_capabilities_string(self, capabilities_string):
             desired_capabilities[key.strip()] = value.strip()
 
         return desired_capabilities
+
+
+class CapabilitiesConfig(object):
+
+    def __init__(self, arg):
+        for k in arg:
+            self.__setattr__(k, arg[k])
```

Comment:
I don't understand. This was the only solution I found to return an object. What is the alternative?